src/Symfony/Bridge/Twig/Mime/NotificationEmail.php

```diff
@@ -225,15 +236,17 @@ private function getExceptionAsString(\Throwable|FlattenException $exception): s
      */
     public function __serialize(): array
     {
-        return [$this->context, $this->theme, parent::__serialize()];
+        return [$this->context, $this->theme, $this->rendered, parent::__serialize()];
     }
 
     /**
      * @internal
      */
     public function __unserialize(array $data): void
     {
-        if (3 === \count($data)) {
+        if (4 === \count($data)) {
+            [$this->context, $this->theme, $this->rendered, $parentData] = $data;
+        } elseif (3 === \count($data)) {
             [$this->context, $this->theme, $parentData] = $data;
```

Comment:
Is worth adding a comment as done two lines below or should it be updated to cover this new case?